### PR TITLE
Add `d` and `u` Vim commands to `vim-mode.js`

### PR DESCRIPTION
**Pull request by guitaripod**
Adds quick scrolling actions and sensible defaults. Also removes the keyup event for scrolling as that's not what Vim does.

### CODE REVIEW

File: vim-mode.js [hunk 1]
```diff
@@ -177,19 +177,27 @@ function copyUrlToClipboard () {
...
+    } else if (e.key === 'd') {
+      window.scrollBy(0, quickScrollAmount)
+    } else if (e.key === 'u') {
+      window.scrollBy(0, -quickScrollAmount)
     }
   }
 })
 
 const commandChars = new Set(['f', 'F', 'y', 'g', 'G', 'c'])
-const linkChars = new Set(alphabet)
```
Comment (PalmerAL) [inline vim-mode.js]: Why is this deleted? This gets used on line 251.

**Review by PalmerAL — COMMENTED** [review]
Looks good otherwise. Also, can you document these in the readme?

File: vim-mode.js (hunk 1 above)
Comment (guitaripod) [reply vim-mode.js]: That's an oversight and out of scope diff. My b

**Comment by guitaripod** [comment]
Will do

**Review by PalmerAL — APPROVED** [review]

**Comment by PalmerAL** [comment]
Thanks!
